### PR TITLE
Updating Coding Challenge Readme

### PR DESCRIPTION
**Pull request by aioimo**
CH: https://app.shortcut.com/getkwara/story/26196/improve-frontend-code-challenge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,29 +27,30 @@ Here are 2 visuals for you:
 
 ![The Loan Page, with the Inbox](./assets/inbox-page.png)
 
-The Slider has been already implemented for you in the current repo.
-
-Clone it and follow the instructions to get it up and running.
+The current repo is the the Slider in its current form, developed previously by a junior dev.
```
Comment (fadiktekin) [inline README.md:30]:
```suggestion
The current repo is the Slider in its current form, developed previously by a junior dev.
```

File: README.md [hunk 2]
```diff
@@ -27,29 +27,30 @@ Here are 2 visuals for you:
...
+It is functionally working and making calls to an endpoint that returns Loan data `/api/inbox/loans`. 
+Clone the repo and follow the instructions to get it up and running.
 If everything goes smoothly you will have a local version of the slider visibile on http://localhost:3000/
```
Comment (fadiktekin) [inline README.md:33]: ...version of the slider visibile 

should be 

...version of the slider **visible**

File: README.md [hunk 3]
```diff
@@ -27,29 +27,30 @@ Here are 2 visuals for you:
...
 
-For this exercise we will simulate a real world situation at Kwara.
-The slider is currently working and making calls to an endpoint that returns Loan data `/api/inbox/loans`.
-
 We now have to support a second endpoint that returns data related to the Till, `api/inbox/till`.
-This is your task for today, making sure that the existing `<Inbox />` Component works for both use cases.
+This is the primary task for today, making sure that the existing `<Inbox />` Component works for both use cases. The goal is that the one inbox can show the user both types of messages.
+
+You may notice the server is sporadically unstable, returning an error. We ask that you handle the errors in a reasonable way. Unfortunately, the UX designer is out sick, so the implementation is in your hands how best to do that.
```
Comment (fadiktekin) [inline README.md:38]: Haha I love this "the UX designer is out sick"

**Review by fadiktekin — CHANGES_REQUESTED** [review]

**Comment by nobitagit** [comment]
LGTM

File: server/index.js [hunk 4]
```diff
@@ -1,3 +1,5 @@
+// PLEASE DO NOT CHANGE THIS FILE
```
Comment (nobitagit) [inline server/index.js:1]: 👍

**Review by fadiktekin — APPROVED** [review]

**Pull request merged**